homework.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+    // task1
+// for(let i=0; i<=20;i++){
+//   console.log(i)
+// }
+
+// for(let i=12;i>=-14;i--){
+//   if(i%2!==0){
+//     console.log(i)
+//   }
+// } 
+
+// for(let i=12;i>=-14;i--){
+//   if(i%2==0){
+//     console.log(i)
+//   }
+// }
+
+
+// for(let i=50;i>=20;i--){
+//   if(i%3==0){
+//     console.log(i)
+//   }
+// }
+
+   // task2
+
+// let stri='LaunchCode'
+// let arr= [1, 5, 'LC101', 'blue', 42]
+// let i=arr.map((ind)=>{
+// console.log(ind)
+// })
+ 
+// let arr2=stri.split('').reverse()
+// let ar=arr2.map((index)=>{
+// console.log(index)
+// })
+
+
+   // task3
+//  let bigArr=[2, 3, 13, 18, -5, 38, -10, 11, 0, 104];
```

Comment:
Why to use let in definitions? This relates to all the task in this file.
Please read difference between **let** and **const** and write here in the comment.